_includes/header.html

```diff
@@ -2,7 +2,13 @@
 
   <div class="wrapper">
 
-    <a class="site-title" href="{{ site.baseurl }}/">{{ site.title | escape }}</a>
+    <a class="site-title" href="{{ site.baseurl }}/">
+      {% if site.logo %}
+        <img src="/assets/{{ site.logo }}" class="site-logo" width="150" height="auto">
```

Comment:
AFAIK `jekyll-assets` is not embedded in Jekyll by default and will process `_assets/img/` or `assets/images` but I don't see any reason it could create a conflict. cc/ @envygeeks